cmd/operator-sdk/bundle/validate.go

```diff
@@ -156,13 +183,15 @@ func (c bundleValidateCmd) run() error {
 	manifestsDir := filepath.Join(c.directory, registrybundle.ManifestsDir)
 	results, err := validateBundleContent(logger, manifestsDir)
 	if err != nil {
-		return fmt.Errorf("error validating %s: %v", manifestsDir, err)
-	}
-	if checkResults(results) {
-		return errors.New("invalid bundle content")
+		addBundleValidationErrorToResult(err, &res)
```

Comment:
The following would be better:
```suggestion
		addBundleValidationErrorToResult(fmt.Errorf("error validating content in %s: %v", manifestsDir, err), &res)
```